src/test/cypress/cypress/integration/SoundNotification.spec.js

```diff
@@ -155,8 +157,70 @@ describe('Sound notification test', function () {
             cy.tick(10000);
             sound.checkNumberOfEmittedSoundIs(3);
         });
+
+
+
+    });
+
+    describe('Checking sound and alert when receiving notification for hidden cards', function () {
+        it('Check sound and alert when receiving card hidden by filters', () => {
+            opfab.loginWithUser(user);
+            sound.stubPlaySound();
+
+            opfab.navigateToSettings();
+            settings.clickOnSeverity('alarm');  // set severity alarm to be notified by sound
+            opfab.navigateToFeed();
+            feed.filterByAcknowledgement('ack');
+            feed.checkFilterIsActive();
+
+            sendCardAndCheckSoundAndAlertMessage();
+        });
+
+        it('Check sound and alert when receiving card hidden by timeline', () => {
+            opfab.loginWithUser(user);
+            sound.stubPlaySound();
+
+            opfab.navigateToSettings();
+            settings.clickOnSeverity('alarm');  // set severity alarm to be notified by sound
+            opfab.navigateToFeed();
+
+            // reset ack filter
+            feed.filterByAcknowledgement('notack');
+            feed.checkFilterIsNotActive();
+            // Set timeline to day domain
+            setTimeLineDomain('J');
+            // shift timeline forward by 2 days
+            moveTimelineRight();
+            moveTimelineRight();
+            
+            sendCardAndCheckSoundAndAlertMessage();
+        });
     });
 
+    function sendCardAndCheckSoundAndAlertMessage() {
+        feed.checkNumberOfDisplayedCardsIs(0);
+        sendCardWithSeverityAlarm();
+        sound.checkNumberOfEmittedSoundIs(1);
+        feed.checkNumberOfDisplayedCardsIs(0);
+
+        cy.get('#div-detail-msg').find('span').eq(0).contains('You have received a card hidden by the filters you have activated (Timeline or card feed)');
+        // Check that alert message css class
+        cy.get('of-alert').find('.opfab-alert-business').should('exist');
+        cy.wait(6000);
+        // Check that alert message is not closed automatically after 6 seconds
+        cy.get('#div-detail-msg').should('exist');
```

Comment:
simulate time instead of waiting 6sec using `cy.clock` and  `cy.tick` (See example inside the current test)